reference/hoon-expressions/rune/lus.md

```diff
@@ -135,42 +135,74 @@ Any Hoon expression, `q`, may be used to define the arm computation.
 
 ### +* "lustar"
 
-
-Produce a type constructor arm.
+Used to assign an alias to doors and define virtual arms.
 
 ##### Syntax
 
-Regular: **2-fixed**.
+Regular: **variadic**.
 
 ```hoon
-+*  p=term  [q=term]  r=spec
++*  a=term  b=hoon
+    c=term  d=hoon
+    ...
+    e=term  f=hoon
 ```
 
-`p` is the arm name, `q` is the name of the argument for the constructor function enclosed in square brackets, and `r` is a structure expression that defines the constructor function.
+`a`, `c`, `e` are arm names and `b`, `d`, `f` are any Hoon expression. Note that unlike all other
+runes with a variable number of arguments, the list of arguments of `+*` does
+not end with a terminating rune.
```

Comment:
> all other runes
> terminating rune

I still feel uncomfortable with calling `++` and `--` and such _runes_. `--` is a terminator (for a rune), `++` is, uh, technically superfluous syntax, but good at clarifying where the arm names/sections in a core are.

Reply:
Yeah, the core arms (`++`, `+*`, `+$`) are not runes, but separators (like the comma in `%=(a b 1, c 2)`). And the terminators (`--`, `==`) are *definitely* not runes.

Reply:
Wow, this is news to me. I am pretty sure this is not made explicit in Hoon school (but I'll check later). And we definitely have these things in the runes portion of the reference section and other spots too. I'm going to make a new issue for this.

Reply:
We used to refer to them as "ascii digraphs" but at some point we just gave up. I agree they are not runes, but calling them ascii digraphs is cumbersome and we also had to document _what_ they did in the cases of the lus ones in particular so it got filed under runes as that is where you're likely to look for that information.

Reply:
Alright, I changed it to talk about "`++` arms" instead, and changed terminating runes->terminators.